src/main/java/seedu/duke/BusRoute.java

```diff
@@ -0,0 +1,66 @@
+package seedu.duke;
+
+import seedu.duke.exceptions.CustomException;
+import seedu.duke.exceptions.ExceptionType;
+
+public class BusRoute {
```

Comment:
This class was removed in a previous commit. You might have to update your PR to reflect the new code and make changes accordingly.
PS: There will be more changes required once #28 is merged but those should be minor

Reply:
Alright, will update the code accordingly to the most recent pull request by Yuxin.